release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -659,6 +659,40 @@ The xref:../scalability_and_performance/psap-node-feature-discovery-operator.ado
 [id="ocp-4-8-dev-exp"]
 === Developer experience
 
+[id="ocp-4-8-insights-operator"]
+=== Insights Operator
+
+[id="ocp-4-8-insights-operator-restricted-network"]
+==== Insights Advisor recommendations for restricted networks
+
+In {product-title} 4.8, users operating in restricted networks can gather and upload Insights Operator archives to Insights Advisor to diagnose potential issues. Additionally, users can obfuscate sensitive data contained in the Insights Operator archive before upload.
+
+For more information, see xref:../support/remote_health_monitoring/remote-health-reporting-from-restricted-network.adoc[Using remote health reporting in a restricted network].
+
+[id="ocp-4-8-insights-operator-widget"]
+==== Insights widget improvements
+
+In {product-title} 4.8, the Insights widget now directs the user to more information in {cloud-redhat-com} when no issues are found. 
+
+[id="ocp-4-8-insights-operator-data-collection-enhancements"]
+==== Insights Operator data collection enhancements
+
+In {product-title} 4.8, the Insights Operator collects the following additional information:
+
+* Non-identifiable cluster workload information to find known security and version issues.
+* The `MachineHealthCheck` and `MachineAutoscaler` definitions.
+* The `virt_platform` and `vsphere_node_hw_version_total` metrics.
+* Improved identification of stack trace messages in container pod logs.
+* The `75-worker-sap-data-intelligence` and `99-sdi-generated-containerruntime` `MachineConfigs` related to SAP Data Intelligence.
```

Comment:
And I would omit this line as well. This is actually our problem so sorry, but I found out that the gatherer is not called in the 4.8 so we will need to fix & backport it.